src/test/java/myfirstproject/service/impl/PrepareDataImplTest.java

```diff
@@ -0,0 +1,33 @@
+package myfirstproject.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.HashMap;
+import java.util.Map;
+import myfirstproject.model.Fruit;
+import myfirstproject.service.PreparingData;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class PrepareDataImplTest {
```

Comment:
let's also check a case when we call `preparingData.prepare(new StringBuilder(), Collections.emptyMap())`
also, why we pass `StringBuilder` as a parameter to `prepare` method?
can we create instance inside `prepare`?